testapps/test_pillowtop/tests/test_case_pillow.py

```diff
@@ -1,40 +1,45 @@
-from django.test import TestCase
 import uuid
+from unittest.mock import patch
+
+from django.test import TestCase
 
 from elasticsearch.exceptions import ConnectionError
 
-from corehq.apps.es import CaseES
+from pillow_retry.models import PillowError
+from pillowtop.es_utils import initialize_index_and_mapping
+
+from corehq.apps.es import CaseES, CaseSearchES
 from corehq.elastic import get_es_new
 from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
-from corehq.form_processor.tests.utils import FormProcessorTestUtils, run_with_all_backends
+from corehq.form_processor.tests.utils import (
+    FormProcessorTestUtils,
+    run_with_all_backends,
+)
 from corehq.pillows.mappings.case_mapping import CASE_INDEX_INFO
-from corehq.util.elastic import delete_es_index, ensure_index_deleted
-from corehq.util.test_utils import trap_extra_setup, create_and_save_a_case
-from pillowtop.es_utils import initialize_index_and_mapping
+from corehq.pillows.mappings.case_search_mapping import CASE_SEARCH_INDEX_INFO
+from corehq.util.elastic import ensure_index_deleted
+from corehq.util.test_utils import create_and_save_a_case, trap_extra_setup
 from testapps.test_pillowtop.utils import process_pillow_changes
 
 
 class CasePillowTest(TestCase):
-
     domain = 'case-pillowtest-domain'
 
-    @classmethod
-    def setUpClass(cls):
-        super(CasePillowTest, cls).setUpClass()
-        cls.process_case_changes = process_pillow_changes('DefaultChangeFeedPillow')
-        cls.process_case_changes.add_pillow('case-pillow', {'skip_ucr': True})
-
     def setUp(self):
         super(CasePillowTest, self).setUp()
+        self.process_case_changes = process_pillow_changes('DefaultChangeFeedPillow')
+        self.process_case_changes.add_pillow('case-pillow', {'skip_ucr': True})
         FormProcessorTestUtils.delete_all_cases()
         with trap_extra_setup(ConnectionError):
             self.elasticsearch = get_es_new()
             initialize_index_and_mapping(self.elasticsearch, CASE_INDEX_INFO)
-        delete_es_index(CASE_INDEX_INFO.index)
```

Comment:
I think this change caused another failure since probably other tests aren't clearing the index:
```
======================================================================
FAIL: testapps.test_pillowtop.tests.test_case_pillow:CasePillowTest.test_case_pillow
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 281, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 253, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 267, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 249, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_case_pillow.py", line 50, in test_case_pillow
    self.assertEqual(1, results.total)
AssertionError: 1 != 22
```

Guessing we should just add the line back?

Reply:
Yeah that sounds right. https://github.com/dimagi/commcare-hq/pull/25482/commits/2a7695776a63f20849936014a2bbc785fcb555f9